client/src/pages/Favourites.js

```diff
@@ -14,38 +14,44 @@ const FavouriteDiv = styled.div`
   max-width: 600px;
 `;
 
-const Favourites = () => {
+const Favourite = () => {
   return (
     <FavouriteDiv>
       <PageHeadline>Suche</PageHeadline>
       <Standort>
         <li details>
           Suchergebniss
```

Comment:
Suchergbnis...nur ein s :D
aber ist ja eh nur der Platzhalter ;)

Reply:
ich glaub ich habs kontinuierlich falsch geschrieben in allen seiten 😅`

Reply:
Ist geändert 👍